api/src/main/scala/io/codegalaxy/api/data/InfoData.scala

```diff
@@ -5,7 +5,8 @@ import play.api.libs.json._
 case class InfoData(numberOfQuestions: Int = 0,
                     numberOfPaid: Int = 0,
                     numberOfLearners: Int = 0,
-                    numberOfChapters: Int = 0)
+                    numberOfChapters: Int = 0,
+                    numberOfTheory: Option[Int] = Option.empty)
```

Comment:
I think `None` is shorter then `Option.empty`.

Reply:
Done